### PR TITLE
Added the ability to fetch applications by user

### PR DESCRIPTION
**Pull request by aschade92**
Added two new function, one which fetches all apps fo the organization 

- GetAllApps: fetches all apps for the organization
- GetAppsForUser: fetches all apps for the userID that gets passed in

Couldn't figure out a way to fetch all of the apps at once and then relate them back to users, so we have to fetch the apps individually by user

### CODE REVIEW

**Comment by aschade92** [comment]
https://app.clubhouse.io/intello/story/3227/add-support-to-okta-sdk-to-read-from-apps-user-endpoint

**Comment by alon7** [comment]
Can you setup circleci on this repo?

**Review by alon7 — APPROVED** [review]

**Comment by aschade92** [comment]
@alon7 Yeah sure, i can add it in another pr

**Comment by alon7** [comment]
@aschade92 cool so branch out of this one, add it, we merge it back and then we merge this
Does that work?

**Comment by aschade92** [comment]
Yep!

File: Gopkg.toml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
+
+
+[[constraint]]
+  name = "github.com/google/go-querystring"
```
Comment (alon7) [inline Gopkg.toml:29]: why this constraint?

**Pull request merged**